src/main/java/edu/ksu/canvas/attendance/controller/CourseConfigurationController.java

```diff
@@ -57,6 +77,7 @@ public ModelAndView classSetup(@PathVariable String sectionId,
 
         CourseConfigurationForm courseConfigurationForm = new CourseConfigurationForm();
         courseService.loadIntoForm(courseConfigurationForm, selectedSection.getCanvasCourseId());
+        sectionService.loadIntoForm(courseConfigurationForm, selectedSection.getCanvasCourseId());
         page.addObject("courseConfigurationForm", courseConfigurationForm);
         page.addObject("selectedSectionId", selectedSection.getCanvasSectionId());
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "selectedSectionId" 3 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.ome.ksu.edu/coding_rules#rule_key=squid%3AS1192)